cmd/siac/hostcmd.go

```diff
@@ -283,6 +283,17 @@ RPC Stats:
 			currencyUnits(totalRevenue))
 	}
 
+	// if wallet is locked print warning
+	walletstatus := new(api.WalletGET)
+	walleterr := getAPI("/wallet", walletstatus)
+	if err != nil {
+		die("Could not get wallet status. A working wallet is needed in order to operate your host. Error:", walleterr)
```

Comment:
I'm not sure if `die` is appropriate here; a printed warning (like below) is probably sufficient